File: 251001/Lashkin/src/Core/Domain/Entities/News.cs
namespace Domain.Entities;

public class News : BaseEntity
{
    public long UserId { get; set; }
    public string Title { get; set; } = null!;
    public string Content { get; set; } = null!;
    public DateTime Created { get; set; }
    public DateTime Modified { get; set; }
    
    public virtual User User { get; set; } = null!;
    public virtual Notice Notice { get; set; } = null!;
    public virtual ICollection<Label> Labels { get; set; } = new List<Label>();
}

File: 351004/Leshok/Lab6/models/News.js
// models/News.js
class News {
  constructor(id, title, content, creatorId) {
    this.id = id;
    this.title = title;
    this.content = content;
    this.creatorId = creatorId;
    this.created = new Date().toISOString();
    this.modified = new Date().toISOString();
  }
}
module.exports = News;